### ISSUE TITLE
[3.3] Test failure on old canonical code

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#5913]
@CarsonF can you have a chat with me soon-ish about this:

```
There was 1 failure:

1) Bolt\Tests\Configuration\ResourceManagerTest::testCustomRequest
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'http://test.dev/bolt/test/location'
+'http://bolt.dev/bolt/test/location'

[snip]/tests/phpunit/unit/Configuration/ResourceManagerTest.php:238
```

**Comment by CarsonF** [comment]
It passes for me

**Comment by GawainLynch** [comment]
Fails on Travis & here … which makes me wonder what is different between your set up, and mine/Travis as this is the second issue (c.f. PHP 7.1) that you're not seeing and Travis and I are.

**Comment by CarsonF** [comment]
Linux and OSX?

**Comment by GawainLynch** [comment]
Fixed in #5935
